osfoffline/application/background.py

```diff
@@ -47,6 +47,3 @@ def stop(self):
         RemoteSyncWorker().stop()
         LocalSyncWorker().stop()
 
-        OperationWorker().join()
```

Comment:
Why were the join statements removed here? Particularly asking in context of `LocalSyncWorker` (which may not finish shutting down the watchdog observer instantly; see for example comment here: https://github.com/abought/OSF-Offline/blob/91a819116c93fa0991c957008e415acbd0b1a8b8/osfoffline/sync/local.py#L36-L36).